media-api/app/controllers/MediaApi.scala

```diff
@@ -120,10 +131,32 @@ object MediaApi extends Controller with ArgoHelpers {
     }
   }
 
-  def deleteImage(id: String) = Authenticated {
-    Notifications.publish(Json.obj("id" -> id), "delete-image")
-    // TODO: use respond
-    Accepted.as(ArgoMediaType)
+
+  val ImageCannotBeDeleted = respondError(MethodNotAllowed, "cannot-delete", "Cannot delete persisted images")
```

Comment:
Sticking to the convention in this file. I think it originated in these being objects, similarly to `MethodNotAllowed` etc.

I don't feel particularly strong about it, just seems to make sense to me given how they're used. I don't mind changing but could do outside this PR.